yarn.lock

```diff
@@ -5354,24 +5354,24 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@metamask/eth-snap-keyring@npm:^12.0.0, @metamask/eth-snap-keyring@npm:^12.1.0":
-  version: 12.1.0
-  resolution: "@metamask/eth-snap-keyring@npm:12.1.0"
+"@metamask/eth-snap-keyring@npm:^12.0.0, @metamask/eth-snap-keyring@npm:^12.1.1":
```

Comment:
I didn't released those deps bumps on `core` yet, but since we're not bumping the major, the version range is still compatible!